### PR TITLE
improve README [add a) roadmap, b) community meeting]

### PR DESCRIPTION
**Pull request by andife**
Signed-off-by: Andreas Fehlner <fehlner@arcor.de>

### Description
Add information about the a) roadmap process, b) Community Meetings 

### Motivation and Context
The LFX tool from the linux foundation (https://insights-v2.lfx.linuxfoundation.org/lfai-onnx/overview) uses some regexp to check if some information is avaiable. In this way, on the one hand, more checks will be successful, and on the other hand, the information will be easier to find.

### CODE REVIEW

**Review by gramalingam — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -40,12 +40,29 @@ Check out our [contribution guide](docs/CONTRIBUTING.md) to get started.
 If you think some operator should be added to ONNX specification, please read
 [this document](docs/AddNewOp.md).
 
+# Community meeting
+
+The schedules of the regular meetings of the Steering Committee, the working groups and the SIGS can be found at https://wiki.lfaidata.foundation/pages/viewpage.action?pageId=18481196
```
Comment (jcwchen) [inline README.md:45]:
```suggestion
The schedules of the regular meetings of the Steering Committee, the working groups and the SIGs can be found [here](https://wiki.lfaidata.foundation/pages/viewpage.action?pageId=18481196).
```

File: README.md [hunk 2]
```diff
@@ -40,12 +40,29 @@ Check out our [contribution guide](docs/CONTRIBUTING.md) to get started.
...
+
+
+A large Community Day is held at least 1 time per year. Up to now the following community day took place:
+
+2020.04.09 https://wiki.lfaidata.foundation/display/DL/LF+AI+Day+-ONNX+Community+Virtual+Meetup+-+Silicon+Valley+-+April+9
+2020.10.14 https://wiki.lfaidata.foundation/display/DL/LF+AI+Day+-+ONNX+Community+Workshop+-+October+14
+2021.03.24 https://wiki.lfaidata.foundation/pages/viewpage.action?pageId=35160391
+2021.10.21 https://wiki.lfaidata.foundation/pages/viewpage.action?pageId=46989689
+2022.06.24 https://wiki.lfaidata.foundation/display/DL/ONNX+Community+Day+-+June+24
```
Comment (jcwchen) [inline README.md:54]: Please note that the document shows L48-L54 are squeezed. Perhaps we can use "*" to list each of them.

File: README.md [hunk 3]
```diff
@@ -40,12 +40,29 @@ Check out our [contribution guide](docs/CONTRIBUTING.md) to get started.
 If you think some operator should be added to ONNX specification, please read
 [this document](docs/AddNewOp.md).
 
+# Community meeting
```
Comment (prasanthpul) [inline README.md:43]:
```suggestion
# Community meetings
```

File: README.md (hunk 1 above)
Comment (prasanthpul) [inline README.md:45]:
```suggestion
The schedules of the regular meetings of the Steering Committee, the working groups and the SIGS can be found at [http://onnx.ai/calendar](http://onnx.ai/calendar)
```

File: README.md [hunk 4]
```diff
@@ -40,12 +40,29 @@ Check out our [contribution guide](docs/CONTRIBUTING.md) to get started.
...
+
+The schedules of the regular meetings of the Steering Committee, the working groups and the SIGS can be found at https://wiki.lfaidata.foundation/pages/viewpage.action?pageId=18481196
+
+
+A large Community Day is held at least 1 time per year. Up to now the following community day took place:
```
Comment (prasanthpul) [inline README.md:48]:
```suggestion
Community Meetups are held at least once a year. Content from previous community meetups are at:
```

**Review by prasanthpul — CHANGES_REQUESTED** [review]
left a couple revisions

File: README.md (hunk 2 above)
Comment (andife) [reply README.md]: Thanks, I changed it

File: README.md (hunk 3 above)
Comment (andife) [reply README.md]: Thanks, I changed it

**Review by gramalingam — APPROVED** [review]

**Review by prasanthpul — APPROVED** [review]

**Pull request merged**